src/dolphin/os/OSAlloc.c

```diff
@@ -233,7 +233,8 @@ size_t OSCheckHeap(OSHeapHandle heap)
         CHECK(922, cell->next == NULL || cell->next->prev == cell)
         CHECK(923, MINOBJSIZE <= cell->size)
         CHECK(924, OFFSET(cell->size, ALIGNMENT) == 0)
-        CHECK(925, cell->next == NULL || (char*) cell + cell->size < (char*) cell->next)
+        CHECK(925, cell->next == NULL ||
+                       (char*) cell + cell->size < (char*) cell->next)
```

Comment:
How does the newline not mess up the assert string? And shouldn't the setting you added to .clang-format prevent this from getting formatted?

Reply:
I noticed the same thing. No idea, but it matches so I'm not complaining.